# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/doc/Debugger/Debugger.Frame.md
# Commit: ea1bb2f0fd39
# Full Hash: ea1bb2f0fd39a412480bd777ba944b276a54006a
# Author: Logan Smyth <loganfsmyth@gmail.com>
# Date: 2019-12-09 09:50:39
# Regressor Bug: 1600204
# File Overlap Count: 1
# Description:
#   Bug 1600204 - Part 5: Support .environment on suspended generator frames. r=jimb
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D55235
# ==============================================================================

diff -r 39d1ae1352b3 -r ea1bb2f0fd39 js/src/doc/Debugger/Debugger.Frame.md
--- a/js/src/doc/Debugger/Debugger.Frame.md	Fri Dec 06 02:30:30 2019 +0000
+++ b/js/src/doc/Debugger/Debugger.Frame.md	Sun Dec 08 23:53:27 2019 +0000
@@ -247,7 +247,7 @@
 that do not represent the evaluation of debuggee code, like calls
 non-debuggee functions, host functions or `"debugger"` frames.
 
-Accessing this property will throw if `.onStack == false`.
+Accessing this property will throw if `.terminated == true`.
 
 ### `callee`
 The function whose application created this frame, as a debuggee value,